client/src/lib/tracking.ts

```diff
@@ -37,10 +37,14 @@ const storeCampaignInfosInCookie = () => {
       .map((key) => key + "=" + utmData[key])
       .join("&");
 
+    // expire utmz cookie after 1 day
+    let expires = new Date();
+    expires.setFullYear(expires.getFullYear() + 1);
```

Comment:
it seems that the expiration date is set to 1 year here. Is it expected?

Reply:
Oops no, sorry about that.  I'll clean that up.

Reply:
Should be good now @matthieu-fesselier ! 

```shell
❯ node
Welcome to Node.js v20.13.1.
Type ".help" for more information.
> var expires = new Date()
undefined
> expires
2024-07-16T15:43:27.516Z
> expires.setTime(expires.getTime() + 24 * 60 * 60 * 1000)
1721231007516
> expires
2024-07-17T15:43:27.516Z
> 
```